### ISSUE TITLE
[Bug][BetterRoleColors] Plugin cannot start.

### ISSUE THREAD

**Issue by wastie** [rauenzi/BetterDiscordAddons#144]
When i launch Discord, it says this 
https://i.imgur.com/2xMncbe.png
i tried reinstalling it , it didn't work even reinstalled it multiple times.

this is my plugins folder
https://i.imgur.com/QMTdB0E.png

**Comment by silent** [comment]
I also get this.

**Comment by LePandaFou77** [comment]
Same error, I think is Discord break BetterDiscord by modifying css in yesterday's update.

(Sry for my bad english)

**Comment by wastie** [comment]
can you send me an updated one? or and older version?

**Comment by Arashiryuu** [comment]
@wastie there currently is no updated one, and an older version will also be broken. The breaking change was not from the plugin but rather from Discord itself meaning until the plugin receives an update any and all versions will not work.

tl;dr - just wait for fix, old versions bad too.

**Comment by Kallyopae** [comment]
For me after the latest discord update, it doesn't even show up in plugins.

**Comment by rauenzi** [comment]
I do not support DiscordSource, please stop using it. I am leaving this issue opening but updating the title.

**Comment by rauenzi** [comment]
This has been fixed in latest update.
